apps/redi-connect/src/App.tsx

```diff
@@ -38,7 +39,9 @@ const NormalRediConnect = () => {
         <Router history={history}>
           <Suspense fallback={<Loader loading={true} />}>
             <QueryParamProvider ReactRouterRoute={Route}>
-              <Routes />
+              <ActiveFilterContextProvider>
+                <Routes />
+              </ActiveFilterContextProvider>
```

Comment:
I do agree this looks a little weird here! Let's talk on Slack why we really need a Context here because this is the only way when we want to have a Provider around the Applications page.